nva-datamodel-java/src/main/java/no/unit/nva/model/associatedartifacts/file/AdministrativeAgreement.java

```diff
@@ -37,8 +38,10 @@ public AdministrativeAgreement(
         @JsonProperty(LICENSE_FIELD) Object license,
         @JsonProperty(ADMINISTRATIVE_AGREEMENT_FIELD) boolean administrativeAgreement,
         @JsonProperty(PUBLISHER_AUTHORITY_FIELD) boolean publisherAuthority,
-        @JsonProperty(EMBARGO_DATE_FIELD) Instant embargoDate) {
-        super(identifier, name, mimeType, size, license, administrativeAgreement, publisherAuthority, embargoDate);
+        @JsonProperty(EMBARGO_DATE_FIELD) Instant embargoDate,
+        @JsonProperty(RIGTHTS_RETENTION_STRATEGY) RightsRetentionStrategy rightsRetentionStrategy) {
+        super(identifier, name, mimeType, size, license, administrativeAgreement, publisherAuthority,
```

Comment:
It occurs to me that we need to migrate existing data to a sensible default…we need to think about this

Reply:
I assumed that since RightsRetentionStratey is a new field in the datamodel, all existing data  that don't have a RightsRetentionStrategy set, will get the default, i.e. NullRightsRetentionStrategy during migrating.